packages/cli/lib/commands/repo/releases.js

```diff
@@ -65,11 +65,14 @@ class RepoReleases extends base_1.default {
             progress.tick(results.repository.releases.nodes.length);
             releases.push(...results.repository.releases.nodes);
         } while (results.repository.releases.pageInfo.hasNextPage);
-        if (format === "JSON") {
+        if (format === "JSONL") {
```

Comment:
Not something we should worry about with this one, but the fact that you had to do this a bunch of times makes me think we should refactor this 😄